server/bin/state/config/pbench-server.cfg

```diff
@@ -29,6 +29,13 @@ documentroot = %(unittest-dir)s/var-www-html
 # Expand the results role's tasks to add pbench-move-unpacked
 tasks = pbench-dispatch, pbench-unpack-tarballs, pbench-move-unpacked, pbench-copy-sosreports, pbench-edit-prefixes, pbench-index
 
+[pbench-server-backup]
+endpoint_url = %(unittest-dir)s/pbench-local/s3-backup
+bucket_name = testbucket
+access_key_id = ACCESS_KEY_ID
+secret_access_key = SECRET_ACCESS_KEY
+debug_unittest = True
```

Comment:
@portante pointed out that there should be no separate  `debug_unittest` option in this section: everybody should use the canonical option in the [pbench-server] section. N.B. That requires a change in `lib/s3config/__init__.py`.

Reply:
It also requires a change in `pbench-index.py` and changes to the `pbench-server.cfg` files for the unittests.